data/letsketch-wp9620.tablet

```diff
@@ -0,0 +1,34 @@
+# LetSketch
+# WP9620
+#
+# HID Descriptor:
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/171
+#
+# Button Map:
+#    *-----------------------*
+#    |                       |
+#  A |                       |
+#  B |                       |
+#  C |        TABLET         |
+#  D |                       |
+#  E |                       |
+#    |                       |
+#    *-----------------------*
+#
+[Device]
+Name=LetSketch LetSketch
+ModelName=WP9620
+DeviceMatch=usb:6161:4d15
+PairedIDs=
+Class=Bamboo
+Width=8.27 # 210mm
+Height=5.51 # 140mm
```

Comment:
These need to be ints please, just round them to whatever the box/marketing says. no-one really uses those anyway.